src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileFunction.java

```diff
@@ -95,6 +97,8 @@ public class ModuleFileFunction implements SkyFunction {
 
   public static final Precomputed<Map<String, ModuleOverride>> MODULE_OVERRIDES =
       new Precomputed<>("module_overrides");
+  public static final Precomputed<Map<String, PathFragment>> INJECTED_REPOSITORIES =
```

Comment:
let's use ImmutableMap everywhere (unless the values can be null, which doesn't seem to be the case)